src/main/java/com/leets/commitatobe/domain/commit/service/ExpService.java

```diff
@@ -78,20 +71,34 @@ public void calculateAndSaveExp(String githubId) {
 		user.updateTotalCommitCount(totalCommitCount);
 		user.updateTodayCommitCount(todayCommitCount);
 
+		updateUserRankings();
+		commitRepository.saveAll(commits);//변경된 커밋 정보 데이터베이스에 저장
+		userRepository.save(user);//변경된 사용자 정보 데이터베이스에 저장
+	}
+
+	private int updateConsecutiveDays(LocalDateTime lastCommitDate, LocalDateTime commitDate,
+		int currentConsecutiveDays) {
+		if (lastCommitDate != null && commitDate.isEqual(lastCommitDate.plusDays(1))) {
+			return currentConsecutiveDays + 1;
+		}
+		if (lastCommitDate == null) {
+			return 0;
+		}
+		return currentConsecutiveDays;
```

Comment:
`lastCommitDate == null` 이라는 조건이 먼저 처리 된다면, 위에 있는 `lastCommitDate != null`은 필요 없는 코드 같습니다!

```suggestion
		if (lastCommitDate == null) {
			return 0;
		}
		if (commitDate.isEqual(lastCommitDate.plusDays(1))) {
			return currentConsecutiveDays + 1;
		}
		return currentConsecutiveDays;
```

Reply:
아 early return만 사용한다고 생각해서 그 부분을 미쳐 생각하지 못했네요... 바로 수정하였습니다!!